### ISSUE TITLE
NSUnknownKeyException using UITextView

### ISSUE THREAD

**Issue by palleraccio** [schibsted/layout#107]
Hi everyone,

I've encountered an issue adding a simple `UITextView` from a xml file. 

This is the exception that is thrown:

`Terminating app due to uncaught exception 'NSUnknownKeyException', reason: '[<UITextView 0x7fd26f830c00> valueForUndefinedKey:]: this class is not key value coding-compliant for the key lineBreakMode.`

and this is the UITextView definition:

```
<UITextView
    width="auto"
    height="auto"
    dataDetectorTypes="phoneNumber, link"
    isEditable="false">
    Hello world!
</UITextView>
```

I've tried to pass in several ways the value for the lineBreakMode property, but without success.
Has anyone encountered a similar issue?

**Comment by nicklockwood** [comment]
@palleraccio thanks for reporting. I'll put up a fix asap. In the meantime, using the `text` property instead seems to work:

        <UITextView
            width="auto"
            height="auto"
            dataDetectorTypes="phoneNumber, link"
            isEditable="false"
            text="Hello world!"
        />

**Comment by nicklockwood** [comment]
@palleraccio fixed in version 0.6.18

**Comment by palleraccio** [comment]
@nicklockwood ..perfect! I've tried your fix and it works like a charm! Thanks :-)
